### PR TITLE
Improve UI of Founder and Investor pages

### PR DESCRIPTION
**Pull request by miladsoft**

### CODE REVIEW

File: src/Angor/Client/Components/FounderProjectItem.razor [hunk 1]
```diff
@@ -1,49 +1,144 @@
...
                 if (InvestmentRequests)
                     return;
-                
+
                 InvestmentRequests = true;
                 StateHasChanged();
             });
     }
+
+    public string StripHtmlTags(string input)
```
Comment (dangershony) [inline src/Angor/Client/Components/FounderProjectItem.razor:97]: Such methods should go in some helper class ( I guess they are not unique to this page)
So in the next PR please move them to their own class

File: src/Angor/Client/Pages/Browse.razor [hunk 2]
```diff
@@ -501,25 +501,47 @@ else
         {
             return string.Empty;
         }
+
+        input = Regex.Replace(input, @"<script.*?>.*?</script>", string.Empty, RegexOptions.IgnoreCase | RegexOptions.Singleline);
```
Comment (dangershony) [inline src/Angor/Client/Pages/Browse.razor:505]: same here all this go in a helper class

File: src/Angor/Client/Components/FounderProjectItem.razor (hunk 1 above)
Comment (miladsoft) [reply src/Angor/Client/Components/FounderProjectItem.razor:97]: ok .

File: src/Angor/Client/Pages/Investor.razor [hunk 3]
```diff
@@ -30,141 +30,179 @@
...
+                </span>
+                <div class="h-100 ms-3">
+                    <h5 class="mb-0 font-weight-bolder">
+                        Invested Portfolio
+                    </h5>
+                </div>
+            </div>
+            <div class="d-flex align-items-center justify-content-end w-100 w-lg-auto mt-3 mt-lg-0">
+                <button class="btn btn-border w-100 w-lg-auto me-3 mb-lg-0" @onclick="GetProjectsAndUpdateAsync">
+                    <i class="@(RefreshBalanceTriggered ? "rotate-icon" : "")">
```
Comment (dangershony) [inline src/Angor/Client/Pages/Investor.razor:49]: ah brilliant

File: src/Angor/Client/Pages/Investor.razor [hunk 4]
```diff
@@ -334,12 +372,12 @@
...
         var investorNostrPubKey = investorNostrPrivateKey.PubKey.ToHex()[2..];
-        
+
         var investorProject = new InvestorProject
-        {
-            TransactionId = investment.TransactionId,
-            AmountInvested = investment.TotalAmount,
-        };
+            {
+                TransactionId = investment.TransactionId,
```
Comment (dangershony) [inline src/Angor/Client/Pages/Investor.razor:378]: I often see change just for spaces perhaps we should agree all of us on how we specify spaces, this is clutter for code reviewing

File: src/Angor/Client/Pages/View.razor [hunk 5]
```diff
@@ -583,30 +610,60 @@
...
             await Js.InvokeVoidAsync("window.open", url, "_blank");
         }
         else
         {
-            // Handle the case where pubkey is null or empty
             notificationComponent.ShowNotificationMessage("Public key is not available.", 3);
         }
     }
 
+    public string StripHtmlTags(string input)
```
Comment (dangershony) [inline src/Angor/Client/Pages/View.razor:636]: same with this

**Review by dangershony — APPROVED** [review]

**Pull request merged**